.travis.yml

```diff
@@ -11,5 +11,5 @@ python:
 services:
   - mongodb
 
-script: PYMONGO_MUST_CONNECT=1 python setup.py test
+script: PYMONGO_MUST_CONNECT=1 python setup.py test --no_ext
```

Comment:
This is because the tests don't currently work when the C extensions are installed? Don't you have the same issue in Evergreen?

Reply:
Instead of changing the travis or evergreen config, I suggest skipping the tests when the C extensions are installed. Once you get the BSONWriter working with C we can start testing across the board.